_layouts/cask_json.json

```diff
@@ -0,0 +1,95 @@
+---
+---
+{%- assign full_name = page.name | remove: ".json" -%}
+{%- assign name = full_name | remove: "@" | remove: "." | remove: '+' -%}
```

Comment:
Are there any casks with `@` or `+` in their names?

Reply:
I don't believe so. I just pulled this from formula_json in case there were.

Reply:
@cmbernard333 Can you check? If there's none: these can be removed.